### DISCUSSION TITLE
Are there plans for some kind of option to extract songs and game config?

### DISCUSSION THREAD

**Discussion by Anymusxd** [ppy/osu#30500] [Q&A] [answered]
I'm asking this mainly for people that have plans on cleaning up their computer, want to share a pen with their songs to their friend, etc

I'm aware that copying the entire osu!lazer folder and then installing the game and pasting everything works, but maybe not everyone wants everything copied because time or storage and they don't know what folder exactly to copy

**Comment by peppy** [comment, marked as answer]
There's no current plans, no. You may find third party tools which can do this for you though.
